build.sbt

```diff
@@ -10,6 +10,7 @@ lazy val root = (project in file("."))
     scriptedLaunchOpts ++= List("-Xms1024m", "-Xmx1024m", "-XX:ReservedCodeCacheSize=128m", "-XX:MaxMetaspaceSize=256m", "-Xss2m", "-Dfile.encoding=UTF-8"),
     resolvers += Resolver.url("typesafe", url("http://repo.typesafe.com/typesafe/ivy-releases/"))(Resolver.ivyStylePatterns)
   )
+  .enablePlugins(ScriptedPlugin)
```

Comment:
Is that needed? What does it do? I didn't see it at https://www.scala-sbt.org/1.0/docs/Testing-sbt-plugins.html

Reply:
Yeah doesn't compile without it for 1.2, works with 1.1/0.13